capture.py

```diff
@@ -416,6 +421,17 @@ def _isolated_nodes(nodes, panel):
             cmds.isolateSelect(panel, addDagObject=obj)
     yield
 
+@contextlib.contextmanager
+def _maintained_time():
+    """Context manager for preserving (resetting) the time after the context"""
+    from maya import cmds
```

Comment:
I actually wrote it by hand and then I remembered you were doing the same thing in the Quicktime Extractor in Magenta. So I adapted it to look similar and use the same name. Is that what you meant?